actions/stop-all.go

```diff
@@ -20,24 +20,21 @@ import (
 
 //StopAllCommand to stop codewind and project containers
 func StopAllCommand() {
-	containerArr := [2]string{}
+	containerArr := [4]string{}
 	containerArr[0] = "codewind-pfe"
 	containerArr[1] = "codewind-performance"
-	containerArr[0] = "cw-"
-	containerArr[1] = "appsody"
+	containerArr[2] = "cw-"
+	containerArr[3] = "appsody"
 
```

Comment:
Can we also move this to a simpler array declaration:
```
containerArr := []string{
"codewind-pfe", 
"codewind-performance",
"cw-",
"appsody",
}
```
Given the original bug was because we were assigning to indexes manually it would make sense.

Reply:
agreed